core/src/main/java/org/apache/calcite/adapter/jdbc/JdbcSchema.java

```diff
@@ -433,6 +433,17 @@ private static RelDataType sqlType(RelDataTypeFactory typeFactory, int dataType,
                 typeFactory.createSqlType(SqlTypeName.ANY), true);
       }
       return typeFactory.createArrayType(component, -1);
+    case GEOMETRY:
```

Comment:
According to the elements in `SqlTypeName#JDBC_TYPE_TO_NAME` , whether there are any cases that can be matched `GEOMETRY`?